lamindb/curators/__init__.py

```diff
@@ -911,6 +918,296 @@ def __init__(
         self._columns_field = self._var_fields
 
 
+class CatColumn:
+    """Categorical column for `DataFrame`.
```

Comment:
Why not call it `CatColumManager` or something like this?

Reply:
Not sure, because `ColumnCatManager` sounds like it should inherit from `CatManager`.